ansible.cfg

```diff
@@ -1,6 +1,7 @@
 [defaults]
 roles_path = vendor/roles
 force_handlers = True
+vault_password_file = .vault_pass.example
```

Comment:
Shouldn't this be set to `.vault_pass`? It's one less thing to change if users want it enabled.